src/test/java/org/sagebionetworks/web/unitclient/widget/table/modal/fileview/CreateTableViewWizardStep2Test.java

```diff
@@ -153,9 +167,12 @@ public class CreateTableViewWizardStep2Test {
   @Before
   public void before() {
     MockitoAnnotations.initMocks(this);
-    when(mockTableEntity.getId()).thenReturn(ENTITY_ID);
-    when(mockEntityView.getId()).thenReturn(ENTITY_ID);
-    when(mockSubmissionView.getId()).thenReturn(ENTITY_ID);
+    mockTableEntity = new TableEntity();
+    mockEntityView = new EntityView();
+    mockSubmissionView = new SubmissionView();
+    mockTableEntity.setId(ENTITY_ID);
+    mockEntityView.setId(ENTITY_ID);
+    mockSubmissionView.setId(ENTITY_ID);
```

Comment:
We are now passing these to EntityTypeUtils, which checks the class to give us an `EntityType`. That check doesn't work with mocks, so just use "real" objects.